README.md

```diff
@@ -151,18 +151,26 @@ kubectl create -f config/samples/operator_v1alpha1_gatekeeper.yaml
 ```
 
 ## Editing default ExemptNamespaces 
-Go to controllers/config_helper.go. Change exemptNamespaces from line 23
+Go to [controllers/config_helper.go](controllers/config_helper.go). Update `spec.match[].exemptNamespaces` in the `getDefaultConfig` function.
```

Comment:
Is there a field in the Gatekeeper CR that we can tell people using the operator to update?  I'm not sure why we're pointing to the code in the readme.

Reply:
My fist thought is for developers who want to add exempt namespaces (maybe other teams wants to add and open PR).